functionary/ui/templatetags/social_helper.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def find_account(context, provider_id):
```

Comment:
Let's try to remember docstrings on functions / methods.  Type hints are encouraged as well.